src/components/tables/TableConnected.tsx

```diff
@@ -22,40 +22,48 @@ import { ITableHeaderCellState } from './TableHeaderCellReducers';
 import { ITableCompositeState, ITableState } from './TableReducers';
 import { getTableChildComponentId } from './TableUtils';
 
-const mapStateToProps = (state: IReactVaporState, ownProps: ITableOwnProps): ITableCompositeStateProps => {
-  const tableState: ITableState = state.tables[ownProps.id] || {} as ITableState;
-  const filterState: IFilterState = tableState && _.findWhere(state.filters, { id: tableState.filterId });
-  const paginationState: IPaginationState = tableState && _.findWhere(state.paginationComposite, { id: tableState.paginationId });
-  const perPageState: IPerPageState = tableState && _.findWhere(state.perPageComposite, { id: tableState.perPageId });
+export interface IReactVaporStateExtended extends IReactVaporState {
+  [additionalState: string]: any;
```

Comment:
open to a better way to do this...

Reply:
Is `additionalState` used? I wonder if you could simply use `IReactVaporState` and in other projects make the `UAState` extends `IReactVaporState`.

In this case you'll be able to call the function with any store that extends the react-vapor one and the function will use only know properties

Reply:
I put my other comment in the wrong place:
Won't Typescript complain if I pass a state like `UAState` that contains a lot of substates that are not defined in `ReactVaporState`?

Reply:
If UAState extends IReactVaporState it will be ok because every properties that the method use exists in UAState

Reply:
Works perfectly, Thanks!